pkg/util/metrics/metrics_test.go

```diff
@@ -0,0 +1,88 @@
+package metrics
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+)
+
+func TestGetResultByError(t *testing.T) {
+	tests := []struct {
+		name  string
+		error error
+		want  string
+	}{
+		{
+			name:  "testnoerr",
+			error: nil,
+			want:  "success",
+		},
+		{
+			name:  "testerr",
+			error: errors.New("hello,error"),
+			want:  "error",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := GetResultByError(tt.error); got != tt.want {
+				t.Errorf("GetResultByError() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestDurationInMicroseconds(t *testing.T) {
+	tests := []struct {
+		name  string
+		start time.Time
+	}{
+		{
+			name:  "testMicroseconds",
+			start: time.Date(2022, 10, 1, 7, 8, 9, 0, time.Local),
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := DurationInMicroseconds(tt.start)
```

Comment:
I'm not sure that for these functions about time, such a test method is sufficient.

Reply:
I have the same thought, but this function can't get a definite value for comparison. Can you give me some advice? Thanks!

Reply:
/cc @XiShanYongYe-Chang for opinion.

Reply:
Personally, I think it makes no sense to test this function. I suggest removing unused functions.
Hi @my-git9 How do you think?

Reply:
Do you mean we want to delete unused functions from "pkg/util/metrics/metrics.go"? It seems that none of the functions inside were used

Reply:
> Do you mean we want to delete unused functions from "pkg/util/metrics/metrics.go"? 

Yes.

Those two methods seem not used:

https://github.com/karmada-io/karmada/blob/594ad9f44b67d199354ce375c9df43adcb6187e9/pkg/util/metrics/metrics.go#L22-L30

Reply:
Has beed updated, please review again, thx

Reply:
Thanks~ There is a lint error.